src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -95,6 +96,120 @@ public static FunctionPointerMethodSymbol CreateFromSource(FunctionPointerTypeSy
         public static FunctionPointerMethodSymbol CreateFromMetadata(CallingConvention callingConvention, ImmutableArray<ParamInfo<TypeSymbol>> retAndParamTypes)
             => new FunctionPointerMethodSymbol(callingConvention, retAndParamTypes);
 
+        public FunctionPointerMethodSymbol SubstiteParameterSymbols(TypeWithAnnotations substitutedReturnType, ImmutableArray<TypeWithAnnotations> substitutedParameterTypes)
+            => new FunctionPointerMethodSymbol(
+                this.CallingConvention,
+                this.RefKind,
+                substitutedReturnType,
+                this.RefCustomModifiers,
```

Comment:
>this.RefCustomModifiers [](start = 16, length = 23)

I think these are subject for substitution as well. Please add a test.

Reply:
I have no idea what could cause these to be substituted. Can you please show a scenario?

---
In reply to: [406852356](https://github.com/dotnet/roslyn/pull/43041#discussion_r406852356) [](ancestors = 406852356)

Reply:
> I have no idea what could cause these to be substituted. Can you please show a scenario?

I guess, the modifier is a generic type closed over ```T```. When ```T``` is substituted, the modifier should change accordingly.

---
In reply to: [406863326](https://github.com/dotnet/roslyn/pull/43041#discussion_r406863326) [](ancestors = 406863326,406852356)

Reply:
Adding a test is tracked by the tracking issue.

---
In reply to: [406865984](https://github.com/dotnet/roslyn/pull/43041#discussion_r406865984) [](ancestors = 406865984,406863326,406852356)